### ISSUE TITLE
Could you provide some results on those public datasets?

### ISSUE THREAD

**Issue by ZhengkunTian** [tencent-ailab/pika#1]
I'm very interested in this project? Could you provide some results on those public dataset, like aishell-1 or librispeech?

**Comment by cweng6** [comment]
Yes, we are considering an example on aishell 1 or 2. Stay tuned.
